core/eolearn/core/visualizations.py

```diff
@@ -0,0 +1,339 @@
+        :type fill_numeric: float
+        :return: dataframe with dummy data
+        :rtype: geopandas.GeoDataFrame
+        """
+        dataframe = pd.DataFrame(data=blank_timestamps, columns=[self.timestamp_column])
+
+        for column in geodataframe.columns:
+            if column == self.timestamp_column:
+                continue
+            elif column == self.geometry_column:
+                dataframe[column] = dummy_geometry
+            elif column == 'valid':
+                dataframe[column] = False
+            elif geodataframe[column].dtype in (int, float):
+                dataframe[column] = fill_numeric
+            else:
+                dataframe[column] = fill_str
+
+        return dataframe
+
+    def create_dummy_polygon(self, addition_factor):
+        """ Creates geometry/polygon to plot if there is no data (at timestamp)
+
+        :param addition_factor: size of the 'blank polygon'
+        :type addition_factor: float
+        :return: polygon
+        :rtype: shapely.geometry.Polygon
+        """
+        x_blank, y_blank = self.eopatch.bbox.lower_left
+        dummy_geometry = Polygon([[x_blank, y_blank],
+                                  [x_blank + addition_factor, y_blank],
+                                  [x_blank + addition_factor, y_blank + addition_factor],
+                                  [x_blank, y_blank + addition_factor]])
+
+        return dummy_geometry
+
+    def plot_scalar_label(self, feature_type, feature_name):
+        """ Line plot for FeatureType.SCALAR, FeatureType.LABEL
+
+        :param feature_type: type of eopatch feature
+        :type feature_type: FeatureType
+        :param feature_name: name of eopatch feature
+        :type feature_name: str
+        :return: visualization
+        :rtype: holoviews/geoviews/bokeh
+        """
+        data_da = array_to_dataframe(self.eopatch, (feature_type, feature_name))
+        if data_da.dtype == np.bool:
+            data_da = data_da.astype(np.int8)
+        return data_da.hvplot()
+
+    def plot_shapes_one(self, data_gpd, timestamp, crs):
+        """ Plots shapes for one timestamp from geopandas GeoDataFrame
+
+        :param data_gpd: data to plot
+        :type data_gpd: geopandas.GeoDataFrame
+        :param timestamp: timestamp to plot data for
+        :type timestamp: datetime
+        :param crs: in which crs is the data to plot
+        :type crs: sentinelhub.crs
+        :return: visualization
+        :rtype: geoviews
+        """
+        out = data_gpd.loc[data_gpd[self.timestamp_column] == timestamp]
+        return gv.Polygons(out, crs=ccrs.epsg(int(crs.value)))
+
+    def plot_vector_timeless(self, feature_name):
+        """ Plot FeatureType.VECTOR_TIMELESS data
+
+        :param feature_name: name of the eopatch featrue
+        :type feature_name: str
+        :return: visalization
+        :rtype: geoviews
+        """
+        crs = self.eopatch.bbox.crs
+        if crs == CRS.WGS84:
```

Comment:
Same as with comparing `FeatureType` objects it is also nicer to use `is` instead of `==`  for comparing `CRS` objects.

Reply:
changed to `is`